### PR TITLE
Save taskmap's position

### PR DESCRIPTION
**Pull request by IidaKainu**
Position (x, y -coordinates and zoom level) is saved to redux.

Taskmap's position is cleared when project overview is opened or roadmap is selected from the dropdown
- clearing the position on SELECT_CURRENT_ROADMAP was not suitable as it is run in every reload

### CODE REVIEW

File: frontend/src/components/RoadmapSelectorWidget.tsx [hunk 1]
```diff
@@ -62,15 +62,23 @@ export const RoadmapSelectorWidget = () => {
...
+              e.preventDefault();
+              dispatch(roadmapsActions.clearTaskmapPosition());
+              history.push(roadmapPath);
+            }}
+            to={roadmapPath}
+          >
+            {roadmap.name}
+          </Link>
+        );
+      })}
```
Comment (juhoeerola) [inline frontend/src/components/RoadmapSelectorWidget.tsx:81]:
```suggestion
      {roadmaps.map((roadmap) => (
        <Link
          key={roadmap.id}
          className={classes(css.dropItem)}
          onClick={() => {
            dispatch(roadmapsActions.clearTaskmapPosition());
          }}
          to={`${paths.roadmapHome}/${roadmap.id}${paths.roadmapRelative.dashboard}`}
        >
          {roadmap.name}
        </Link>
      ))}
```

The click handler could be used for just dispatching the redux action, the default would then follow the link

Comment (IidaKainu) [reply frontend/src/components/RoadmapSelectorWidget.tsx]: I thought I tried this with the action not dispatching. Looks like it actually works perfectly fine: thank you

**Review by juhoeerola — APPROVED** [review]
👍

**Review by PatricKangasmaki — APPROVED** [review]
Working well, nice job 👍

**Pull request merged**